### ISSUE TITLE
Figure out how to manually add release entries

### ISSUE THREAD

**Issue by jsta** [jsta/jsta#3]
One option would be to create a separate json file that gets appended to the API results.

**Comment by jsta** [comment]
```json
[{"repo": "rjsta", "login": "jsta", "repo_url": "https://github.com/jsta/rjsta", "description": "A wild mix of experimental and classic R functions", "keywords": "rstats", "release": "0.1-4", "published_at": "2021-02-10", "url": "https://github.com/jsta/rjsta/releases/tag/0.1-4"}, {"repo": "wikilake", "login": "jsta", "repo_url": "https://github.com/jsta/wikilake", "description": "Scrape lake metadata tables from Wikipedia \ud83c\udfde", "keywords": "wikipedia, lakes, limnology, cran, rstats", "release": "0.5.0", "published_at": "2021-02-09", "url": "https://github.com/jsta/wikilake/releases/tag/0.5.0"}]
```

**Comment by jsta** [comment]
> I'm not sure whether you're saying you wish to merge the multiple sources or show them separately with a 'flick of a switch'.
If merging, then it would be best to gather the data outside of DataTables. Use jQuery's ajax() function to get all the data, merge it together into a single object, and pass into DataTables's data.

**Comment by jsta** [comment]
Closed by 4c713784993811c77dbf08ebe164d4d923197d25
